libs/core/src/lib/icon-tab-bar/components/icon-tab-bar-filter-type/icon-tab-bar-filter-type.component.html

```diff
@@ -0,0 +1,60 @@
+<ul
+    class="fd-icon-tab-bar__header"
+    role="tablist"
+    fdOverflowItems
+    [isRtl]="isRtl"
+    itemSelector="[data-overflowItem]"
+    [overflowOffset]="100"
+    (overflowChanged)="_recalculateVisibleItems($event)"
+>
+    <li
+        *ngIf="showTabAll"
+        class="fd-icon-tab-bar__item"
+        role="presentation"
+        data-overflowItem
+    >
+        <a (click)="_selectItem(_tabs[0])"
+           [attr.aria-selected]="_tabs[0]?.uId === _selectedUid"
+           class="fd-icon-tab-bar__tab"
+           role="tab"
+        >
+            <div class="fd-icon-tab-bar__container fd-icon-tab-bar__container--filter">
+                <span *ngIf="tabsConfig[0]?.counter" class="fd-icon-tab-bar__filter-counter">{{tabsConfig[0]?.counter}}</span>
+                <span class="fd-icon-tab-bar__filter-label">{{tabsConfig[0]?.label}}</span>
+            </div>
+        </a>
+    </li>
+    <li *ngFor="let item of _tabs | slice:1"
+        [ngClass]="item.cssClasses"
```

Comment:
`[class]` binding is preferred over the `[ngClass]`

https://angular.io/guide/built-in-directives#adding-and-removing-classes-with-ngclass
https://angular.io/guide/attribute-binding#binding-to-multiple-css-classes

But it behaves something differently:
https://github.com/angular/angular/issues/40623
https://hackmd.io/jzDc7hIDTdWtQblv2TbL9A